WordPress/gutenberg
# DataViews: Fix patterns, templates and template parts pagination `z-index`

## What?
Fixes: https://github.com/WordPress/gutenberg/issues/58929

From the issue: 

> "My patterns" pagination is unusable when the view is scrolled down with patterns exactly behind the pagination links. When you click on the pagination, the detail of the pattern behind it opens instead.

This is an issue for template and template parts list too, and this PR fixes that too.

## Testing Instructions
1. Open Appearance > Editor and select Patterns.
2. In My Patterns, create or import patterns until you see the pagination appear at the bottom
3. Scroll the view in the middle, leaving the pagination right on top of a pattern
4. Click on the pagination
5. Pagination controls can be clicked
6. Repeat the above for templates list and template parts list.
